docs/advanced/integration_tests.md

```diff
@@ -0,0 +1,130 @@
+# Integration tests (advanced)
+
+There are several types of Gosoline integration tests, each of them meant to be used for a specific situation. All of them have the same thing in common: the Gosoline `suite` package is responsible for configuring, starting and running an application, then the tests cases are run.
```

Comment:
Okay, but what is important for me here? There are different types... but what types? Why do I need different types of tests? The fact that gosoline starts, configures, and runs the application and test seems quite obvious